src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.lang.String;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class CalculatorTest {
+    Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    @DisplayName("정수 덧셈 테스트")
+    void plus(){
+        assertThat(calculator.plus(2,3)).isEqualTo(5);
```

Comment:
```suggestion
        assertThat(calculator.stringCalculator("2 + 3")).isEqualTo(5);
```
이런식으로 변경해 볼수 있을거 같아요😄

다른 테스트도 동일하게 변경 부탁드려요~